flopy/utils/binaryfile/__init__.py

```diff
@@ -1610,10 +1610,14 @@ def get_ts(self, idx, text=None, times=None, variable="q"):
 
         Parameters
         ----------
-        idx : tuple of ints, or a list of a tuple of ints
-            idx can be (layer, row, column) or it can be a list in the form
-            [(layer, row, column), (layer, row, column), ...].  The layer,
-            row, and column values must be zero based.
+        idx : int, tuple of ints, or list of such
+            Acceptable values depend on grid type:
```

Comment:
Described required syntax for each grid type as requested. But I tentatively reworked this to accept a properly-shaped tuple for the grid type. Having to pass a 3-tuple with dummy values was very awkward and I'm tempted to call this a fix even though it's a breaking change. I don't know of anywhere else this is done that way. Wondering how others feel about this.